code/modules/antagonists/traitor/contractor/items/contractor_pinpointer.dm

```diff
@@ -38,7 +38,7 @@
 /obj/item/pinpointer/crew/contractor/trackable(mob/living/carbon/human/H)
 	var/turf/here = get_turf(src)
 	var/turf/there = get_turf(H)
-	return there && there.z == here.z
+	return here && there && there.z == here.z
```

Comment:
`here` can be null here? That would mean the pinpointer is in null space... or qdeleted and still running/processing

Reply:
A runtime was raised during the TM.. so yes